counting-svg/counting-svg.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: MIT
+// An example of a consumer contract that relies on a subscription for funding.
+pragma solidity ^0.8.17;
+
+// Imports
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "@openzeppelin/contracts/utils/Base64.sol";
+
+contract CountSVG is ERC721, ERC721URIStorage, Ownable {
```

Comment:
Do you need `ERC721` and `ERC721URIStorage`? I think since `ERC721URIStorage` imports `ERC721` you can just import `ERC721URIStorage`, and still use the `ERC721` constructor.